lesson16/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+   "liveServer.settings.port": 5502
+}
```

Comment:
nits
please .gitignore

Reply:
Thank you for your reviews.
Fixed that.
[8e8ac3e](https://github.com/taka0209/js_lesson/pull/18/commits/8e8ac3eae89c9ae5c76850f5482148d3c68d0340)